distribution/client/src/client.js

```diff
@@ -87,9 +73,10 @@ class Client {
 
     this.errorTelemetry.setup();
 
-    setInterval( () => {
+    setInterval(() => {
       /* no-op to keep this process alive */
-    }, 10);
+      this.runUpdates();
+    }, 5000);
```

Comment:
I implemented it on this short timeframe and filed [JENKINS-51272](https://issues.jenkins-ci.org/browse/JENKINS-51272) to restore proper `Pusher` service so that, during integration tests, we can push a `check-updates` command to the client.

I'm starting on that ticket today, time permitting.